### ISSUE TITLE
whiteboard is not getting created for tests

### ISSUE THREAD

**Issue by narasimhan-v** [avocado-framework/avocado#1818]
```
2017-02-24 03:46:06,583 test             L0616 ERROR| Traceback (most recent call last):

2017-02-24 03:46:06,583 test             L0616 ERROR|   File "/usr/local/lib/python2.7/dist-packages/avocado_framework-46.0-py2.7.egg/avocado/core/test.py"
, line 520, in _run_avocado
    genio.write_file(whiteboard_file, self.whiteboard)

2017-02-24 03:46:06,583 test             L0616 ERROR|   File "/usr/local/lib/python2.7/dist-packages/avocado_framework-46.0-py2.7.egg/avocado/utils/genio.p
y", line 170, in write_file
    with open(filename, 'w') as file_obj:

2017-02-24 03:46:06,583 test             L0616 ERROR| IOError: [Errno 2] No such file or directory: 'nvme_results/job-2017-02-24T03.45-ed73b52/test-results
/01-nvmetest.py:NVMeTest.testformatnamespace/whiteboard'

2017-02-24 03:46:06,583 test             L0633 ERROR| ERROR 01-nvmetest.py:NVMeTest.testformatnamespace -> IOError: [Errno 2] No such file or directory: 'n
vme_results/job-2017-02-24T03.45-ed73b52/test-results/01-nvmetest.py:NVMeTest.testformatnamespace/whiteboard'
2017-02-24 03:46:06,584 test             L0620 INFO |
2017-02-24 03:46:06,584 output           L0642 DEBUG| Process Process-1:
2017-02-24 03:46:06,584 output           L0642 DEBUG| Traceback (most recent call last):
2017-02-24 03:46:06,585 output           L0642 DEBUG|   File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
2017-02-24 03:46:06,593 output           L0642 DEBUG|     self.run()
2017-02-24 03:46:06,593 output           L0642 DEBUG|   File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
2017-02-24 03:46:06,594 output           L0642 DEBUG|     self._target(*self._args, **self._kwargs)
2017-02-24 03:46:06,594 output           L0642 DEBUG|   File "/usr/local/lib/python2.7/dist-packages/avocado_framework-46.0-py2.7.egg/avocado/core/runner.p
y", line 319, in _run_test
2017-02-24 03:46:06,595 output           L0642 DEBUG|     instance.run_avocado()
2017-02-24 03:46:06,595 output           L0642 DEBUG|   File "/usr/local/lib/python2.7/dist-packages/avocado_framework-46.0-py2.7.egg/avocado/core/test.py"
, line 621, in run_avocado
2017-02-24 03:46:06,595 output           L0642 DEBUG|     with open(self.logfile, 'r') as log_file_obj:
2017-02-24 03:46:06,595 output           L0642 DEBUG| IOError: [Errno 2] No such file or directory: 'nvme_results/job-2017-02-24T03.45-ed73b52/test-results
/01-nvmetest.py:NVMeTest.testformatnamespace/debug.log'
```

**Comment by ldoktor** [comment]
I wanted to ask for more input, but I think I identified the issue (at least partially). By any chance do you use custom `logs_dir` in the avocado config and `os.chdir` in the test? I was able to reproduce this by:

```diff
diff --git a/etc/avocado/avocado.conf b/etc/avocado/avocado.conf
index 9b767d4..38138e9 100644
--- a/etc/avocado/avocado.conf
+++ b/etc/avocado/avocado.conf
@@ -6,7 +6,8 @@ test_dir = /usr/share/avocado/tests
 # You may override the specific test auxiliary data directory with data_dir
 data_dir = /usr/share/avocado/data
 # You may override the specific job results directory with logs_dir
-logs_dir = ~/avocado/job-results
+#logs_dir = ~/avocado/job-results
+logs_dir = ./
 # You can set a list of cache directories to be used by the avocado test
 # fetch_asset() with 'cache_dirs'. read-only cache directories are also
 # supported.
diff --git a/examples/tests/whiteboard.py b/examples/tests/whiteboard.py
index f248a03..d90aad8 100755
--- a/examples/tests/whiteboard.py
+++ b/examples/tests/whiteboard.py
@@ -18,6 +18,8 @@ class WhiteBoard(Test):
     """
 
     def test(self):
+        import os
+        os.chdir("/tmp")
         data_file = self.params.get('whiteboard_data_file', default='')
         data_size = self.params.get('whiteboard_data_size', default='10')
         if data_file:
```
The thing is we always use `os.abspath` unless custom path is provided in config file. Let me fix this... (and please let me know if this is not your case)

**Comment by narasimhan-v** [comment]
@ldoktor Thanks. This was the exact case here.

```
# grep logs_dir /etc/avocado/avocado.conf 
# You may override the specific job results directory with logs_dir
logs_dir = ~/avocado/job-results
# grep chdir nvmetest.py
        os.chdir("%s/nvme-cli-master" % self.srcdir)
```

Changed to the current directory again in the test, and it worked :+1:

**Comment by ldoktor** [comment]
This should be fixed by the #1819
